test/name_scheme_test.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+
+class NameSchemeTest(unittest.TestCase):
+
+    # The click python_wheel has no name_scheme argument, and is thus using the default value which
+    # is a list.
+    def test_click_is_importable(self):
+        import click
+        self.assertIsNotNone(click.command)
+
+    # The click-log python_wheel has an explicit name_scheme argument which is a string.
+    def test_click_log_is_importable(self):
+        import click_log
+        self.assertIsNotNone(click_log.basic_config)
```

Comment:
Is there risk of these drifting, as they're not linked to the python_wheel defs in any particular way? So if the def changes, these tests might not still be covering what they claim to be?

Reply:
Yes, there's a risk. This isn't the only example of a test relying on a package in //third_party/python, but I'm not sure there's an alternative. I've added a comment on the packages in question to draw attention to this test.